esp/esp/users/forms/user_profile.py

```diff
@@ -310,6 +310,14 @@ def clean(self):
 StudentInfoForm.base_fields['studentrep_expl'].widget.attrs['rows'] = 8
 StudentInfoForm.base_fields['studentrep_expl'].widget.attrs['cols'] = 45
 
+affiliationChoices = (
```

Comment:
Can't put it in models, because of the "Other" issue (see previous comment).  But I did change it to all caps, and also made the same change to several other choice lists.  (By the way, WhatToDoAfterHS doesn't seem to be used.)